src/test/java/io/temporal/worker/StickyWorkerTest.java

```diff
@@ -226,7 +226,7 @@ public void workflowCacheEvictionDueToThreads() {
     int count = 100;
     ActivitiesWorkflow[] workflows = new ActivitiesWorkflow[count];
     WorkflowParams w = new WorkflowParams();
-    w.TemporalSleep = Duration.ofSeconds(1);
+    w.TemporalSleepMillis = 1000;
```

Comment:
For defining time intervals - Do we prefer units on `ints` rather than strongly typed `Duration`?

Reply:
We prefer Duration unless it has to be passed across process boundary and serialized/deserialized. We can add serialization of Duration to the Jackson json serializer, but then it might not play well with Go and other SDKs.